spec/latest/core/index.html

```diff
@@ -630,6 +630,40 @@ <h3>Discovering a Hydra-powered Web API</h3>
       using HTTP Link Headers is typically not necessary as the concepts
       used in responses from the API will dereference to their
       documentation.</p>
+
+    <p>Another case would involve an <i>ApiDocumentation</i> discover from
+      another Api response. There is no restriction on emitting an HTTP
+      <i>Link</i> header on every Api response, thus every response could be
+      used to discover <i>ApiDocumentation</i>.</p>
+
+    <pre class="example nohighlight" data-transform="updateExample"
+         title="Retrieving a Hydra API documentation from another Api response">
+      <!--
+      GET /api/items HTTP/1.1
+      Host: api.example.com
+      Accept: application/ld+json
+
+      ====================================
+
+      HTTP/1.1 200 OK
+      ...
+      Content-Type: application/ld+json
+      ****Link: <http://api.example.com/doc/>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"****
```

Comment:
Do asterisks here somehow affect formatting?

Reply:
It gets greenish in the final spec